### DISCUSSION TITLE
Local docker container Auth is disabled but "You don't have required permission..."

### DISCUSSION THREAD

**Discussion by alexander-newby** [provectus/kafka-ui#3869] [Q&A] [answered]
Following the `Quick start (Demo run)` section of the readme, I started `kafka-ui` in a docker container and saw it say "Authentication is disabled. Access will be unrestricted" in the logs. However, when I visit the page the `Configure new cluster` button is disabled and, when hovering over it, it says "You don't have a required permission to perform this action"

I find this quite confusing - I was expecting and hoping to be able to access the whole UI to test it out using this quick start setup. Can anyone explain to me what I am doing wrong or if this is a bug or something?

In case it matters, I am trying this on an M1 Mac and I have not got Kafka running locally (I was hoping to connect to a remote cluster).

**Comment by Haarolean** [comment, marked as answer]
Hi, it's a UI bug, already fixed and merged. Please try pulling `master`-labeled image.

**Reply by alexander-newby** [reply]
Yes, that works, thank you!
